### ISSUE TITLE
Billing page error

### ISSUE THREAD

**Issue by snide** [withastro/studio-issues#15]
Started a project with one of the templates. I originally skipped the credit screen that I was presented with (which is why I think this failed). The billing page now errors.

> RPCClientError: No payment method found

Guessing that's because I skipped it earlier.

**Comment by ciw1973** [comment]
Yeah, getting the same, and it means there no option to enter billing details if you skip doing so initially.
